microprofile/grpc/core/src/main/java/io/helidon/microprofile/grpc/core/model/AbstractMethodHandlerSupplier.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.microprofile.grpc.core.model;
+
```

Comment:
You should use a flat package structure in Helidon - e.g. each module (maven module) has a single package.
This is to enable proper use of package local visibility (and so we do not need `internal` packages. 
If really needed, move this to a separate module. If not, refactor to a single package.